nornir_infrahub/plugins/tasks/artifact.py

```diff
@@ -4,7 +4,9 @@
 
 def regenerate_host_artifact(task: Task, artifact: str) -> Result:
     node = task.host.data["InfrahubNode"]
-    artifact_node = node._client.get(kind="CoreArtifact", name__value=artifact, object__ids=[node.id])
+    artifact_node = node._client.get(
+        kind="CoreArtifact", definition__artifact_name__value=artifact, object__ids=[node.id]
```

Comment:
What is the reason behind this change?

Reply:
We're passing in the name of the artifact definition, at least in our example. I may have made this change hastily, and I should have updated the `examples/nornir_tasks.py` to pass in the artifact name rather than the artifact definition name.

Reply:
We can change the behavior for sure if that is desirable, but this would be a breaking change that we need to communicate properly